site/components/form/Checkbox.tsx

```diff
@@ -15,11 +16,27 @@ const Checkbox = <T extends object>({
     hideLegend = false,
     checkboxDetail,
     initialErrors = [],
+    reset,
```

Comment:
This attribute allows updating the input using uncontrolled element which can be avoided. Remove this attribute and `ref` references. Remove the useEffect. Update `defaultChecked={item.checked}` to `checked={item.checked}`. 

In `create-disruption.page.tsx` 

- update `checked: validity.disruptionNoEndDateTime === "true"` to `checked: !!validity.disruptionNoEndDateTime`
- update `pageState.inputs.publishNoEndDateTime === "true"` to `!!pageState.inputs.publishNoEndDateTime`

Reply:
this will not work  "true" to checked: !!validity.disruptionNoEndDateTime as "false" will equate to true since it is a string